dataactbroker/handlers/jobHandler.py

```diff
@@ -259,6 +259,8 @@ def addUploadJobs(self,filenames,submissionId,existingSubmission):
         jobsRequired = []
         # Dictionary of upload ids by filename to return to client
         uploadDict = {}
+        d1ValId = None
+        cValId = None
```

Comment:
If I understand the logic correctly, these vars won't be set unless the `filenames` are in a specific order. Is there a way to be more explicit? Maybe we pull the file C and D1 files out of that list and run them first?